codeval.py

```diff
@@ -298,23 +304,147 @@ def unzip(filepath, dir, delete=False):
         debug(f"{filepath} deleted.")
 
 
-@click.command("codeval")
+def upload_assignment_files(path,course):
+    global file_dict
+    if os.path.exists(path) and not os.path.isfile(path):
+        assign_directory = os.listdir(path)
+        if not assign_directory:
+            error("assignmentFiles directory is empty. Exiting!!")
+        else:
+            for file in assign_directory:
+                canvas_folders=course.get_folders()
+                for fol in canvas_folders:
+                    if fol.name == "CodEval":
+                        try:
+                            file_spec=fol.upload(path + '/' + file)
```

Comment:
this is where you should check dry-run and print the message would upload ... if true

Reply:
So it needs to be only here right and  should not be present before creating and editing the assignment right

Reply:
Done